packages/serialization/form/src/formSerializationWriter.ts

```diff
@@ -131,6 +131,9 @@ export class FormSerializationWriter implements SerializationWriter {
 			}
 		}
 	};
+	// placeholder for the method
+	// eslint-disable-next-line @typescript-eslint/no-empty-function
+	public writeCollectionOfEnumValue = <T>(_key?: string, _values?: (T | null | undefined)[]): void => {};
```

Comment:
why do we need placeholder methods here instead of rolling out the implementation directly?

Reply:
This is currently blocking the testing in PR https://github.com/microsoft/kiota-typescript/pull/1578, the serialization interface cant be referenced in tests

Reply:
Going to close this PR, had an issue with testing changes for the unpublished abstraction interface.